x-pack/plugins/observability_onboarding/public/components/app/system_logs/install_elastic_agent.tsx

```diff
@@ -40,7 +40,7 @@ export function InstallElasticAgent() {
   const [elasticAgentPlatform, setElasticAgentPlatform] =
     useState<ElasticAgentPlatform>('linux-tar');
 
-  const datasetName = 'elastic-agent';
+  const datasetName = 'system-logs';
   const namespace = 'default';
```

Comment:
Unless we have UI to configure `datasetName` and `namespace` maybe these values can be hardcoded in the route as the saved object is created. what do you think?

Reply:
[76d1277](https://github.com/elastic/kibana/pull/162972/commits/76d1277731a6929e44179595b5fc3526a3171ea2)